public/app/core/utils/fetch.ts

```diff
@@ -93,7 +93,7 @@ export const parseBody = (options: BackendSrvRequest, isAppJson: boolean) => {
     return options.data;
   }
 
-  return isAppJson ? JSON.stringify(options.data) : new URLSearchParams(options.data);
+  return isAppJson ? safeStringifyValue(options.data) : new URLSearchParams(options.data);
```

Comment:
Please add a comment why this is done, it will make people's live easier in the future